### PR TITLE
Feature messaging routes

### PR DESCRIPTION
**Pull request by megangilpin**
Added routes for messaging

### CODE REVIEW

**Review by gary-bains — APPROVED** [review]

File: client/src/pages/ChefSearch.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ function ChefSearch() {
     const location = [city, region, country].join(", ");
     const [radiusKm, setRadiusKm] = useState(100);
     const [chefs, setChefs] = useState([]);
+    console.log(chefs);
```
Comment (ThinkTink) [inline client/src/pages/ChefSearch.js]:
```suggestion
```

File: server/routes/messages.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    })
+);
+
+// Delete conversation by conversation id
+router.delete(
+    "/:id",
+    errorHandelingWrapper(async (req, res) => {
+        const { id } = req.params;
+
+        const conversation = await conversationsController.remove(id);
```
Comment (ThinkTink) [inline server/routes/messages.js:117]: Should we be removing all of the linked messages from the database as well?

**Review by ThinkTink — APPROVED** [review]

File: client/src/pages/ChefSearch.js (hunk 1 above)
Comment (megangilpin) [reply client/src/pages/ChefSearch.js]: man. I really tried to catch all those this time.

File: server/routes/messages.js (hunk 2 above)
Comment (megangilpin) [reply server/routes/messages.js:117]: Yes we should. Right now we don't have the functionality to do it from a front end anyways so we've added a separate ticket to delete the messages.

File: client/src/pages/ChefSearch.js (hunk 1 above)
Comment (ThinkTink) [reply client/src/pages/ChefSearch.js]: :smile: I have a checklist I personally run on all my code before pushing to a PR, and it includes doing a find all for console.log. I always catch some

**Pull request merged**